cmd/image-builder/main.go

```diff
@@ -430,6 +468,7 @@ operating systems like Fedora, CentOS and RHEL with easy customizations support.
 	manifestCmd.Flags().String("ostree-url", "", `OSTREE url`)
 	manifestCmd.Flags().Bool("use-librepo", true, `use librepo to download packages (disable if you use old versions of osbuild)`)
 	manifestCmd.Flags().Bool("with-sbom", false, `export SPDX SBOM document`)
+	manifestCmd.Flags().String("registrations", "", `filename of a registrations file with e.g. subscription details`)
```

Comment:
I wonder if it makes sense to generalize the 2nd input JSON file into something more generic. What if we decide to add another input, would that be a 3rd input config file? What I am trying to say, if we called this I dont know `--metadata` and in the file we had `subscriptions` key, we could freely expand it with other fields later on. Not sure, just an idea.

Reply:
My gut-feeling is that starting more targeted is better (YAGNI and all that). I think if we need more we will either do the stacked blueprints with ephemeral data or move to --metadata, and keep --registrations as a alias for --metadata.